src/main/scala/uk/gov/nationalarchives/tdr/api/graphql/fields/FFIDMetadataFields.scala

```diff
@@ -35,8 +35,8 @@ object FFIDMetadataFields {
       fileExtensionMismatch: Option[Boolean] = Some(false)
   )
 
-  case class FFIDMetadataInputMatches(extension: Option[String] = None, identificationBasis: String, puid: Option[String])
-  case class FFIDMetadataMatches(extension: Option[String] = None, identificationBasis: String, puid: Option[String])
+  case class FFIDMetadataInputMatches(extension: Option[String] = None, identificationBasis: String, puid: Option[String], fileExtensionMismatch: Option[Boolean] = Some(false))
```

Comment:
I would add the additional optional param for the "format name" as well. As it sounds like downstream processes will need this aswell: `... formatName: Option[String] = "" ...`